src/Three.js

```diff
@@ -156,5 +156,8 @@ export { Curve } from './extras/core/Curve.js';
 export { ShapeUtils } from './extras/ShapeUtils.js';
 export { SceneUtils } from './extras/SceneUtils.js';
 export { WebGLUtils } from './renderers/webgl/WebGLUtils.js';
+export { WEBGL_CONSTANTS } from './renderers/webgl/WebGLConstants.js';
```

Comment:
I don't really like the need to create a new instance just to query a constant. I would prefer to access it directly:
```js
THREE.WebGLConstants.POINTS;
```

The same for the `fromGL` and `toGL` that I believe too that they should be part of  `WebGLUtils`
```js
THREE.WebGLUtils.fromGL(gl.POINTS)
THREE.WebGLUtils.toGL(THREE.FrontSide)
```